README.md

```diff
@@ -39,11 +43,14 @@ Before you get started, you'll need the following installed and set up:
 * [Java (JDK) 7](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
 * [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files](http://www.oracle.com/technetwork/java/javase/downloads/jce-7-download-432124.html)
   * Install under $JAVA_HOME/jre/lib/security
-* [Maven 3.0.4+](http://maven.apache.org/download.html)    See [installation instructions](http://maven.apache.org/download.html#Installation).
+* [Maven 3.0.5+](http://maven.apache.org/download.html)    See [installation instructions](http://maven.apache.org/download.html#Installation).
 * [MySQL 5.1.x](http://dev.mysql.com/downloads/mysql/5.1.html#downloads)
-* [Eclipse Juno](http://www.eclipse.org/downloads/)
-  * [egit plugin](http://www.eclipse.org/egit/)
-  * [m2eclipse plugin](http://www.eclipse.org/m2e/download/)
+* Application Server (Below are Supported Servers)
+  * [GlassFish v3.1.2.2]
+(https://glassfish.java.net/downloads/3.1.2.2-final.html)
+  * [WebSphere Enterprise 8.5.5.3](http://www.ibm.com/developerworks/downloads/ws/wasdevelopers/)
+  * [WebLogic 12c (12.1.1)](http://www.oracle.com/technetwork/middleware/ias/downloads/wls-main-097127.html)
+  * [JBoss 7.1.1 Final](http://jbossas.jboss.org/downloads/)
 * [Apache Ant v1.7.1](http://ant.apache.org/)
```

Comment:
In CONNECT 4.4 installation instructions wiki, we are specifying v1.7.1, for consistency purpose leaving the version as v1.7.1